### DISCUSSION TITLE
Displaying Not Found page after installation

### DISCUSSION THREAD

**Discussion by xiNeRoar** [shlinkio/discussions#74] [Help wanted] [answered]
#### How Shlink is set-up

* Shlink Version: stable tag
* How do you serve Shlink: Self-hosted Docker image on synology nas

#### Summary
![image](https://user-images.githubusercontent.com/66216186/163695812-b1032870-082d-4201-8928-5746a516adf4.png)
this is the env variable comes with the docker and I set it up correctly as the document instructed;
![image](https://user-images.githubusercontent.com/66216186/163695842-c06d9a37-93b4-4787-805e-25d27401a682.png)
However, when I go to the instance port, it shows the below message and I couldn't find a document to resolve this problem.
![image](https://user-images.githubusercontent.com/66216186/163695851-448ba311-15c5-49f4-995f-345484537e2d.png)

**Comment by acelaya** [comment, marked as answer]
Yes, that's the expected behavior. Now you need to generate an API key:

![Screenshot_20220417-072929](https://user-images.githubusercontent.com/2719332/163701895-148b3906-d67a-4c3e-a6d2-05700099e9e9.png)

Once you have it, you have to use some tool to interact with Shlink. The official one is https://app.shlink.io

You can learn more about it here https://shlink.io/documentation/shlink-web-client/

**Reply by xiNeRoar** [reply]
Thank you so much! so if i didn't get it wrong, what I have installed is simply the server but not a web client?

**Reply by acelaya** [reply]
Exactly https://shlink.io/documentation/advanced/shlink-architecture/

**Reply by xiNeRoar** [reply]
sorry but i still didn't get it...is the "web-client" a admin panel which the "shlink" itself is face to public URL shortener? i am a bit confuse...

**Reply by acelaya** [reply]
Yes. Everything is explained in the URL above.
